ops/ops/interface_kube_control/model.py

```diff
@@ -64,5 +126,16 @@ class Data(BaseModel):
     port: Json[int] = Field(alias="port")
     sdn_ip: Optional[str] = Field(default=None, alias="sdn-ip")
     registry_location: str = Field(alias="registry-location")
-    taints: Optional[Json[List[Taint]]] = Field(alias="taints")
-    labels: Optional[Json[List[Label]]] = Field(alias="labels")
+    taints: Optional[Json[List[Taint]]] = Field(alias="taints", default=None)
+    labels: Optional[Json[List[Label]]] = Field(alias="labels", default=None)
+
+    def ca_certificate(self, model: ops.Model) -> Optional[bytes]:
```

Comment:
IMO, I think this naming looks better. Unless you'd prefer to convert it to a property.
```suggestion
    def get_ca_certificate(self, model: ops.Model) -> Optional[bytes]:
```